wikichange/src/content/content.js

```diff
@@ -127,19 +150,22 @@ const renderSlider = async (creationDate) => {
         curRevisionId = (await fetchRevisionFromDate(title, now))[0];
         oldRevisionId = (await fetchRevisionFromDate(title, date))[0];
 
+        // Change the revision context box
+        const revisionContext = document.getElementById("revisionContext");
+        revisionContext.innerHTML = `Compared to the <a href=${(getRevisionPageLink(curRevisionId, oldRevisionId))} target="_blank">${(await fetchRevisionFromDate(title, date))[1].toLocaleDateString().slice(0, 10)}</a> version, the current page has 30 differences highlighted below`;
         highlightRevisionBetweenRevisionIds(title, curRevisionId, oldRevisionId);
-        revisionButton.disabled = false;
+        // revisionButton.disabled = false;
     });
 
-    revisionButton.addEventListener("click", async function (ev) {
-        try {
-            window.open(getRevisionPageLink(curRevisionId, oldRevisionId), "_blank");
-        } catch (err) {
-            console.error(
-                `Error getting revision link between revision ids for inputs title:${title} curRevisionId:${curRevisionId} oldRevisionId:${oldRevisionId}\nError: ${err}`
-            );
-        }
-    });
+    // revisionButton.addEventListener("click", async function (ev) {
+    //     try {
+    //         window.open(getRevisionPageLink(curRevisionId, oldRevisionId), "_blank");
+    //     } catch (err) {
+    //         console.error(
+    //             `Error getting revision link between revision ids for inputs title:${title} curRevisionId:${curRevisionId} oldRevisionId:${oldRevisionId}\nError: ${err}`
+    //         );
+    //     }
+    // });
```

Comment:
should bring this back.

Reply:
I am keeping the code for the button so if we decide to choose the button I can easily reverse the change.

Reply:
that works.

Reply:
on second thought, should bring revision button back^^